extension/asapauthextension/README.md

```diff
@@ -0,0 +1,36 @@
+# ASAP Client Authentication Extension
+
+This extension provides [Atlassian Service Authentication Protocol](https://s2sauth.bitbucket.io/) (ASAP) client 
```

Comment:
For my own education, do you have a resource with the differences between it and OAuth 2?